components/ElectronicsDropOffCard.js

```diff
@@ -11,7 +11,10 @@ function ElectronicsDropOffCard() {
           "Any organization or business around the world can help families get access to technology. Put a donation box in your office for used laptops, tablets, and smartphones. Volunteers will pick up the items periodically, fix them up, and provide them to families in need. You can optionally be involved with refurbishment and placement too if you'd like."
         }
       </CardParagraph>
-      <CardAddress label="Email us to learn more at:">
+      <CardAddress
+        label="Email us to learn more at:"
+        style={{ marginTop: 'auto' }}
```

Comment:
@LowLifeArcade Reworked all cards, `space-y-3` class [interferes to use different top space for the last element](https://tailwindcss.com/docs/space#limitations). Because of it limitation, I've changed layout to `grid` and it looks like, that it works for all of the cards.

Reply:
@LowLifeArcade Feel free to tell me any ideas to improve it 🙂